shell/common/node_bindings.cc

```diff
@@ -530,10 +530,15 @@ void NodeBindings::Initialize(v8::Local<v8::Context> context) {
 
   // Parse and set Node.js cli flags.
   std::vector<std::string> args = ParseNodeCliFlags();
+
+  // V8::EnableWebAssemblyTrapHandler can be called only once or it will
+  // hard crash. We need to prevent Node.js calling it in the event it has
+  // already been called.
+  node::per_process::cli_options->disable_wasm_trap_handler = true;
```

Comment:
Should this be `true` or some kind of "is renderer" check as I assume the main process + utility process's don't have this set up?

Reply:
We always want it - if you look at the diff you'll see it was true before but we patched it in as an env option - joyee merged a PR to accomplish the same goal